### ISSUE TITLE
Error when using in Moodle 3.4

### ISSUE THREAD

**Issue by heather-williams** [jleyva/moodle-block_configurablereports#97]
When trying to view reports in a course we see the following error:

```
get_role_users() without specifying one single roleid needs to be called prefixing role assignments id (ra.id) as unique field, you can use $fields param for it.
line 3619 of /lib/accesslib.php: call to debugging()
line 45 of /blocks/configurable_reports/components/conditions/usersincurrentcourse/plugin.class.php: call to get_role_users()
line 352 of /blocks/configurable_reports/report.class.php: call to plugin_usersincurrentcourse->execute()
line 386 of /blocks/configurable_reports/report.class.php: call to report_base->elements_by_conditions()
line 71 of /blocks/configurable_reports/viewreport.php: call to report_base->create_report()
```

**Comment by maguila1** [comment]
Hello heather-williams,

Did you solve this problem?? I have the same issue (same Moodle version and same plugin) and I don't know how to solve it.

By the way, I obtain this error only when I add the condition "Users in current report course".. 

Thanks in advance

**Comment by GirlieNinja** [comment]
Hello @maguila1 

It's been a while since I reported this, but I don't recall finding a solution. I do think it was with the same condition of "Users in current report course".

Sorry, wish I was more helpful!

Thanks!

**Comment by maguila1** [comment]
No problem @heather-williams.  Thank you for your reply

**Comment by sarjona** [comment]
Thanks @heather-williams and @maguila1 for reporting this.

It has been fixed in the commit 217de7c7c14d827bc8c077646135b2822e58b1de and it will be included soon in the new plugin version.
